package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
d2-ui 2.16.7 is the last d2-ui version. d2-ui/d2 versions are not synced in their repo, they have d2-ui=26.1.7, d2=26.1.9. I updated to the last one.